ckanext/iati/logic/csv_action.py

```diff
@@ -179,7 +180,12 @@ def json(self):
         _org_data = PublishersListDownload._get_publisher_data()
         for org in _org_data:
             if org.Group.state == 'active' and int(org.package_count) > 0:
-                json_data.append(OrderedDict(list(zip(self._headers, self._prepare(org)))))
+                ordered_dict = OrderedDict(zip(self._headers, self._prepare(org)))
+                for key, value in ordered_dict.items():
```

Comment:
This would make sense to do in the _prepare method, I think.

Actually, I'm really unclear how that can end up as bytes, since it should be an array?